73-ballan-wrasse/technical_specification.md

```diff
@@ -0,0 +1,70 @@
+# Access Request Management Improvements (Ballan Wrasse)
+
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+## Scope
+
+### Outline
+
+The goal of this epic is to improve the management of access requests by giving
+data stewards control over the validity period and enriching the data access request with existing information (e.g., the corresponding DAC) and additional data entered
+by the data stewards (e.g., reason for rejecting a request, internal notes, ticket ID).
+
+### Included/Required
+
+- Data stewards should be able to modify the requested validity period
+  when the request is approved.
+- Data stewards should be able to add a note to the requester
+  (e.g., reason for denial of an access request or modified validity period).
+- Data stewards should be able to add a ticket ID and internal notes.
+- The data presented to the data steward in the access request manager
+  and sent in the notification to the data steward should be enriched
+  with existing information about the request and the dataset (e.g., DAC).
+- When work packages are created, the validity period of the work package
+  access token should be shown to the user.
+
+### Not included
+
+- Collecting additional context data for the request (e.g., affiliation).
+
+## Affected services
+
+- Access Request Service:
+  - Allow changing the validity period in the PATCH request.
```

Comment:
Should we also provide users the possibility of changing their access request access period too?

Reply:
They can already do that (within certain limits) when they create the request. But from then on, we should not allow further changes since that would mean that the data stewards and DAC would need to be informed about the change as well, which would complicate things a lot. Instead, any further communication should happen via the help desk.